diff --git a/etc/ly/config.ini b/etc/ly/config.ini
--- a/etc/ly/config.ini
+++ b/etc/ly/config.ini
@@ -73,7 +73,7 @@ brightness_up_cmd = /usr/bin/brightnessctl -q s +10%
 brightness_up_key = F6
 
 # Erase password input on failure
-clear_password = false
+clear_password = true
 
 # Format string for clock in top right corner (see strftime specification). Example: %c
 # If null, the clock won't be shown
@@ -167,7 +167,7 @@ margin_box_v = 1
 min_refresh_delta = 5
 
 # Set numlock on/off at startup
-numlock = false
+numlock = true
 
 # Default path
 # If null, ly doesn't set a path
